packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -17,10 +17,12 @@ export function useFrequentlyBoughtTogether<TObject>({
   recommendClient,
   threshold,
   transformItems: userTransformItems,
+  initialState,
 }: GetFrequentlyBoughtTogetherProps<TObject>) {
-  const [result, setResult] = useState<GetRecommendationsResult<TObject>>({
-    recommendations: [],
-  });
+  const initialResults = initialState ?? { recommendations: [] };
```

Comment:
We should align the "results" and "state" naming. We seem to use `initialState` and `initialResults` interchangeably.

Reply:
@marialungu Any updates on this?

Reply:
yeah, i'll include it with the next refactors